test/spec/modules/adkernelBidAdapter_spec.js

```diff
@@ -565,7 +565,7 @@ describe('Adkernel adapter', function () {
 
   describe('adapter configuration', () => {
     it('should have aliases', () => {
-      expect(spec.aliases).to.have.lengthOf(12);
+      expect(spec.aliases).to.be.an('array').that.is.not.empty;
```

Comment:
This will ease new alias adding to a person not familiar with javascript development (some of our clients have struggled to add a new alias because of this explicit check). Moreover, the alias list is a kind of constant, so it's a bit strange to check a constant value in a unit test. With this change, it's become more logical as we only check the adaptor interface contract fulfillment.